package.json

```diff
@@ -10,7 +10,7 @@
     "build-storybook": "build-storybook",
     "lint": "yarn eslint && stylelint src",
     "eslint": "eslint src stories .storybook",
-    "test:ci": "yarn typecheck && yarn test && yarn lint",
+    "test:ci": "yarn test && yarn lint",
```

Comment:
We don't need to `yarn typecheck` separately here now as testing now does type-checking.